### PR TITLE
Work on #8

### PR DESCRIPTION
**Pull request by monicachew**
Support Nightly 32 and 33 as a single graph. I think the easiest way to support other channels is to have a little "Nightly/Aurora" selector at the top of the page, since for any given channel there won't be more than one version per build-date.

### CODE REVIEW

**Comment by monicachew** [comment]
@mozkeeler This is live at http://people.mozilla.org/~mchew/test_pinning_dashboard/

File: index.html [hunk 1]
```diff
@@ -1,4 +1,8 @@
 <title>Public Key Pinning Dashboard</title>
+<center>
+<button onclick="makeGraphsForChannel(nightlyVersions)">Nightly</button>
+<button onclick="makeGraphsForChannel(auroraVersions)">Aurora</button>
```
Comment (monicachew) [inline index.html]: Ugly, but it works. We should probably have some sort of indicator to show which graph is active.

Comment (mozkeeler) [inline index.html]: That would be nice. Maybe style the buttons differently depending on which is active? Or just have a label at the top? Another option is to have a drop-down menu.

**Comment by monicachew** [comment]
I added some terrible button styling.

File: dashboard.js [hunk 2]
```diff
@@ -27,52 +51,171 @@ function print(line) {
   document.querySelector('#output').textContent += line + "\n";
 };
 
+function changeView(channel) {
```
Comment (mozkeeler) [inline dashboard.js:54]: What if this were more like:

```
document.querySelector("#" + currentChannel).setAttribute("style", "background-color:white");
currentChannel = channel;
document.querySelector("#" + currentChannel).setAttribute("style", "background-color:#90ed7d");
makeGraphsForChannel(currentChannel);
```

File: dashboard.js [hunk 3]
```diff
@@ -27,52 +51,171 @@ function print(line) {
...
+      function(histogramEvolution) {
+        histogramEvolution.each(function(date, histogram) {
+          var data = histogram.map(function(count, start, end, index) {
+            return count;
+          });
+          // Skip dates with fewer than 1000 submissions
+          // Failure = 0, success = 1
+          if (data[0] + data[1] > minVolume) {
+            date.setUTCHours(0);
+            //print("Measure: " + measure + " version: " + version +
```
Comment (mozkeeler) [inline dashboard.js]: Is this debugging info still necessary?

**Comment by mozkeeler** [comment]
I like the styling :)
Those are my only comments at the moment - other than that I think this is ready to go.

**Pull request merged**